code/tests/checkpoint_example.cpp

```diff
@@ -162,13 +162,15 @@ class CheckpointExample : public BaseTestCase {
     if (stat_old_res < 0 && errno_old == ENOENT &&
         stat_new_res < 0 && errno_new == ENOENT) {
       test_res->SetError(DataTestResult::kFileMissing);
+      test_res->error_description = "test file has completely disappeared";
```

Comment:
Can we mention which file has disappeared? So is the file we are testing for always the same?

Reply:
This error is if the file does not exist at either the old or the new file path (the file is completely gone). I have updated the error string to note the file paths it is checking for the file.